### PR TITLE
Adds limit to create addresses

### PR DESCRIPTION
**Pull request by cmgustavo**
Note: Require latest version of BWS

### CODE REVIEW

File: src/js/services/bwsError.js [hunk 1]
```diff
@@ -93,6 +93,9 @@ angular.module('copayApp.services')
           case 'INVALID_BACKUP': 
             body = gettextCatalog.getString('Wallet seed is invalid');
             break;
+          case 'MAIN_ADDRESS_GAP_REACHED':
+            body = gettextCatalog.getString('You cannot generate more until you register any activity');
```
Comment (matiu) [inline src/js/services/bwsError.js]: Empty addresses limit reached. New addresses cannot be generated.

**Comment by matiu** [comment]
note that this breaks copay with the current installed version of BWS.

**Comment by cmgustavo** [comment]
Yes. Please, don't merge yet.

File: src/js/services/bwsError.js (hunk 1 above)
Comment (cmgustavo) [inline src/js/services/bwsError.js]: Thanks.

**Comment by cmgustavo** [comment]
Updated with latest version of `angular-bitcore-wallet-client`

**Comment by cmgustavo** [comment]
Updated

**Pull request merged**